layouts/shortcodes/keps-data.html

```diff
@@ -0,0 +1,81 @@
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.css">
+
+<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.js"></script>
+<script type="text/javascript">
+  $(document).ready( function () {
+    $('#keps-table').DataTable({
+      lengthMenu: [
+        [10, 25, 50, -1],
+        [10, 25, 50, 'All'],
+      ],
+    });
+});
+</script>
+
+<div>
+  <table id="keps-table" class="table table-hover mt-2 col-md-12 table-sm">
+    <thead class="thead-light">
+      <tr>
+        <th scope="col" class="col-md-1">KEP Number</th>
+        <th scope="col" class="col-md-3">Title</th>
+        <th scope="col" class="col-md-2">SIG</th>
+        <th scope="col" class="col-md-2">Author</th>
+        <th scope="col" class="col-md-1">Created</th>
+        <th scope="col" class="col-md-1">Last Updated</th>
+        <th scope="col" class="col-md-1">Milestone</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{ $keps := getJSON "https://storage.googleapis.com/k8s-keps/keps.json" }}
+      {{ range $index, $data := $keps }}
```

Comment:
I think you can merge these two in one line, unless I missed something. 

Example: https://github.com/kubernetes/website/pull/35228/files#diff-dd3aae73f0f6ad9c85ca442d99c36734b43e4950f105ad5517e7f879abc02af4R9

Reply:
I'm fine with either approach.